core/src/main/resources/default-main.conf

```diff
@@ -1,3 +1,22 @@
+
+# // #!#
```

Comment:
What are these comments used for?

Reply:
Just for my personal reference to find all code changes fast. I will remove them on next iteration.